Tests/RxCocoaTests/UIScrollView+RxTests.swift

```diff
@@ -170,6 +170,53 @@ extension UIScrollViewTests {
         XCTAssertTrue(didScrollToTop)
         subscription.dispose()
     }
+
+    func testDidEndScrollingAnimation() {
+        // direct call `scrollViewDidEndScrollingAnimation`
+        autoreleasepool {
+            let scrollView = UIScrollView()
+            var didEndScrollingAnimation = false
+            
+            let subscription = scrollView.rx.didEndScrollingAnimation.subscribe(onNext: {
+                didEndScrollingAnimation = true
+            })
+            
+            XCTAssertFalse(didEndScrollingAnimation)
+            
+            scrollView.delegate!.scrollViewDidEndScrollingAnimation!(scrollView)
+            
+            XCTAssertTrue(didEndScrollingAnimation)
+            subscription.dispose()
+        }
+        
+        // change contentOffset with animation
+        autoreleasepool {
```

Comment:
@kzaher 
Thank you for pointing out..! 🙇 
I removed `autoreleasepool` closure and fix test code. (added `completed` check)